Formula/jdep.rb

```diff
@@ -0,0 +1,104 @@
+class Jdep < Formula
+  desc "Analyze Java .class files and produce Makefile dependencies."
+  homepage "http://www.fudco.com/software/jdep.html"
+  url "http://www.fudco.com/software/jdep-1.3.tar.gz"
+  sha256 "3dd925ae2523cca59e4aaa7cbb8daa28927b40520f0f8dccead91d9edea1dc65"
+
+  patch :DATA
+
+  def install
+    system "make"
```

Comment:
We'll accept formulae without a `make install`, but it's not particularly difficult to write it into a `Makefile` that already exists & makes life easier for users even outside of package management, so is a "nice thing to have" when upstreams are willing.